src/scripts/create_jentoft.py

```diff
@@ -0,0 +1,180 @@
+"""Create the Jentoft linguistic acceptability dataset."""
+
+import pandas as pd
+from datasets import Dataset, DatasetDict, Split
+from huggingface_hub import HfApi
+from requests import HTTPError
+
+
+def main():
+    """Create the Jentoft linguistic acceptability dataset and upload to HF Hub."""
+    dataset_urls = {
+        "val": "https://raw.githubusercontent.com/matias-jjj/matias_master/refs/heads/main/norgec_dataset/ask_exp_dev.tsv",
+        "test": "https://raw.githubusercontent.com/matias-jjj/matias_master/refs/heads/main/norgec_dataset/ask_exp_test.tsv",
+        "train": "https://raw.githubusercontent.com/matias-jjj/matias_master/refs/heads/main/norgec_dataset/ask_exp_train.tsv",
+    }
+
+    full_val_df = prepare_dataset(dataset_url=dataset_urls["val"])
+    full_test_df = prepare_dataset(dataset_url=dataset_urls["test"])
+    full_train_df = prepare_dataset(dataset_url=dataset_urls["train"])
+
+    # Remove overlapping texts between splits
+    full_train_df, full_val_df, full_test_df = remove_overlapping_samples(
+        full_train_df, full_val_df, full_test_df
+    )
+
+    assert isinstance(full_train_df, pd.DataFrame)
+    assert isinstance(full_val_df, pd.DataFrame)
+    assert isinstance(full_test_df, pd.DataFrame)
+
+    val_size = 256
+    test_size = 2048
+    train_size = 1024
+
+    val_df = sample_dataset(full_df=full_val_df, size=val_size)
+    test_df = sample_dataset(full_df=full_test_df, size=test_size)
+    train_df = sample_dataset(full_df=full_train_df, size=train_size)
+
+    # Collect datasets in a dataset dictionary
+    dataset = DatasetDict(
+        train=Dataset.from_pandas(train_df, split=Split.TRAIN, preserve_index=False),
+        val=Dataset.from_pandas(val_df, split=Split.VALIDATION, preserve_index=False),
+        test=Dataset.from_pandas(test_df, split=Split.TEST, preserve_index=False),
+        full_train=Dataset.from_pandas(
+            full_train_df, split=Split.TRAIN, preserve_index=False
+        ),
+        full_val=Dataset.from_pandas(
+            full_val_df, split=Split.VALIDATION, preserve_index=False
+        ),
+        full_test=Dataset.from_pandas(
+            full_test_df, split=Split.TEST, preserve_index=False
+        ),
```

Comment:
Since these are not used anywhere, I'd remove them here.

Feel free to upload the full dataset elsewhere, since these ScandEval datasets are not really meant to be publicly available anyway.

```suggestion
```